tests/phpunit/record_test.php

```diff
@@ -128,10 +128,12 @@ public function test_can_get_values() {
     /**
      * Test throw coding exception if request invalid property.
      *
-     * @expectedException coding_exception
-     * @expectedExceptionMessage Requested invalid property
+     * @expectException coding_exception
```

Comment:
Should we remove those lines at all?

Reply:
Yes. those lines removed. Thank you @dmitriim